packages/block-editor/src/components/block-patterns-list/README.md

```diff
@@ -0,0 +1,32 @@
+# Block Patterns List
+
+The `BlockPatternList` component makes a list of the different registered block patterns. It uses the `BlockPreview` component to display a preview for each block pattern.
+
+For more infos about blocks patterns, read [this](https://make.wordpress.org/core/2020/07/16/block-patterns-in-wordpress-5-5/).
+
+![Block patterns sidebar in WordPress 5.5](https://make.wordpress.org/core/files/2020/09/blocks-patterns-sidebar-in-wordpress-5-5.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders a block patterns list.
+
+```jsx
+import { BlockPatternList } from '@wordpress/block-editor';
+
+const MyBreadcrumb = () => (
+	<BlockPatternList
+		onClickPattern={ onClick }
```

Comment:
@ntsekouras I was wondering if it is necessary to add the `onClickPattern={ onClick }` to this code sample.

Reply:
Also, I was wondering if there could be ways I can make these code samples more detailed. Til now, for the packages I documented, I basically added a const with a simple use case of the component.

I know these packages component are not intended to be uses by third parties, but was just wondering.

Reply:
This component requires three props 

```
shownPatterns={ shownBlockPatterns }
blockPatterns={ shownBlockPatterns }
onClickPattern={ onSelectBlockPattern }
```

Ideally, we pass a valid prop and explain what these props are about. (Some READMEs have a props section for the explanations)

Reply:
_Still having to find what the props do exactly_. On my list for next week.